client/src/pages/TaskFormPage/TaskFormPage.js

```diff
@@ -20,21 +91,41 @@ const TaskFormPage = () => {
             header={"Task Title"}
             type={"text"}
             placeholder={"Task Name"}
+            onChange={(e) => updateTaskFormData(e, "title")}
+            value={taskFormData.title}
           />
           <InputBox
             header={"Location"}
             type={"text"}
             placeholder={"Location Name"}
+            onChange={(e) => updateTaskFormData(e, "location")}
+            value={taskFormData.location}
           />
           <InputBox
             header={"Description"}
             type={"text"}
             placeholder={""}
             expanded={true}
+            onChange={(e) => updateTaskFormData(e, "description")}
+            value={taskFormData.description}
+          />
+          <p className={styles.inputHeader}>Start</p>
+          <DateSelector
+            showTime={true}
+            selectedDate={new Date(taskFormData.startDate)}
+            onDateChanged={(newDate) => updateTaskDate(newDate, "startDate")}
+          />
+
+          <p className={styles.inputHeader}>End</p>
+          <DateSelector
+            showTime={true}
+            selectedDate={new Date(taskFormData.endDate)}
```

Comment:
Refer to the comment on line 115 for this date selector as well

Reply:
Changed accordingly also.